db/re/item_db_equip.yml

```
@@ -19603,7 +19605,6 @@ Body:
     Script: |
       bonus bAtkEle,Ele_Holy;
       bonus bDex,4;
-      bonus bMatkRate,15;
       bonus2 bSubRace,RC_Demon,15;
```

Aegis item script give matkrate +15%
```suggestion
      bonus bMatkRate,15;
      bonus2 bSubRace,RC_Demon,15;
```

---

Thanks